pkgs/tools/security/keybase/default.nix

```diff
@@ -12,8 +12,8 @@ buildGoPackage rec {
   src = fetchFromGitHub {
     owner  = "keybase";
     repo   = "client";
-    rev    = "v${version}";
-    sha256 = "1642d11gjgkdklppmm1j3vwc2r3qg9qqw5x07jnqs819i57mr47f";
+    rev    = "8169d666e2a4d920da7025170baccde336fce11d";
```

Comment:
Please also add your explanation from your pull request here as well, so future maintainers know which version to update to.